composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "razorpay/magento",
     "description": "Razorpay Magento 2.0 plugin for accepting payments.",
-    "version": "3.8.5-beta",
```

Comment:
@nikhilsharma1100 If required raise separate PR for version bump. Please avoid including version bump with enhancement OR fix PR.

Reply:
This is no version bump in this PR. It is showing this change because I raised PR from another branch which had this beta version while testing.